File: Hello/mvklingeren/hello.ts
class Greeter{
    private _name:string="";
    constructor(name: string) {
        this._name=name;
    }

    public toString = () : string => {
        return `Hi! (${this._name}), greets from Hello-World`;
    }

}

let helloWorld = new Greeter("Milco");
console.log(helloWorld);

File: Hello/danialrajabi/hello.py
#! /usr/bin/env python3

class HelloWorld:
    def __init__(self, name):
        self.name = name

    def sayhello(self):
        return ('#######################################################\n'
               f'##############   HelloWorld from {self.name}   #############\n'
                '###########       Hacktoberfest 2019        ###########\n'   
                '#######################################################'
                )


    def __repr__(self):
        return 'Hello World'



if __name__ == '__main__':
    danial = HelloWorld(name='danial')
    print(danial.sayhello())
